deploy/osps/default/osp-flatcar.yaml

```diff
@@ -471,6 +471,13 @@ spec:
                 echo -e "[Service]\nEnvironment=\"KUBELET_NODE_IP=${DEFAULT_IFC_IP}\"\nEnvironment=\"KUBELET_HOSTNAME=${FULL_HOSTNAME}\"" > /etc/systemd/system/kubelet.service.d/nodeip.conf
               fi
 
+      - path: /etc/systemd/network/zz-default.network.d/ipv6-fix.conf
+        permissions: 755
+        content:
+          inline:
+            data: |
+              [Network]
+              IPv6AcceptRA=true
       - path: /opt/bin/setup
```

Comment:
Is it possible to configure this [here](https://github.com/kubermatic/operating-system-manager/blob/master/deploy/osps/default/osp-flatcar.yaml#L838)? Have you tried that? 

Just asking coz we have a dedicated section where I think we can configure this. But ofcourse I have never tried your workflow :)

Reply:
Workflow?

Reply:
Idk why you picked "workflow" out of the whole comment and didn't bother to reply on the main question but that's fine :D 

I have never tried the "workflow" that you are following; never worked on fiddling with `IPv6AcceptRA` property for the network service.

Reply:
Just trying to clarify stuff. I thought there was a concept in OSM called workflow that I totally missed.

Reply:
Let us discuss it here https://github.com/kubermatic/machine-controller/pull/1393/files/2d0bab43befb507b91e8e7ef5facc4d7a74a2a18#r937756877 

Whatever we decide there should be applicable here too. I will make the necessary changes once the decision is made.